app/api/helpers/tasks.py

```diff
@@ -388,6 +388,23 @@ def export_speakers_csv_task(self, event_id):
     return result
 
 
+@celery.task(base=RequestContextTask, name='export.speakers.pdf', bind=True)
+def export_speakers_pdf_task(self, event_id):
+    speakers = db.session.query(Speakers).filter_by(event_id=event_id)
```

Comment:
The model name is `Speaker` and not `Speakers`.

Reply:
It should be `speakers = db.session.query(Speaker).filter_by(event_id=event_id)`